plugin/federation/federation_test.go

```diff
@@ -77,6 +77,12 @@ func TestNoEntities(t *testing.T) {
 	require.Len(t, f.Entities, 0)
 }
 
+func TestInterfaces(t *testing.T) {
+	require.Panics(t, func() {
```

Comment:
Can you say more about the purpose of this test?  Are we just looking to verify that we can load a configuration that includes a schema that has interfaces?  Could this be an entity resolver test instead to verify that interfaces work?

Reply:
So this is a negative test to verify that we panic out when attempting to apply @key to an interface. Happy to move this to another PR, but it seems related.